.mergify.yml

```diff
@@ -7,5 +7,8 @@ pull_request_rules:
   conditions:
   - label!=WIP
   - '#approved-reviews-by>=1'
-  - check-success=run tests
+  - check-success=test (3.6)
+  - check-success=test (3.7)
+  - check-success=test (3.8)
+  - check-success=test (3.9)
```

Comment:
Does this just cover the unit tests or will the flake8 ones block a merge?

Reply:
It doesn't look like we've got CI for flake8 yet, looking at the github actions run below... Oh, looking at https://github.com/release-depot/flask-container-scaffold/blob/main/.github/workflows/test.yml flake8 is repeated for every python version.

Reply:
Ah nice catch. Maybe a refinement later would be to pull out the flake8 test and run it only once on the latest python version. That is not needed for this review however.

Reply:
Currently, the flake tests are run with each version of python as part of the same job.  This should probably be separated in the near future, but I think that should be done indepentently of this change.